package.json

```diff
@@ -34,5 +36,8 @@
     "@babel/preset-env": "^7.8.7",
     "babel-jest": "^25.1.0",
     "jest": "^25.1.0"
+  },
+  "engines": {
+    "node": ">=13.0.0"
```

Comment:
Stable Node.js only, please!

Reply:
Yeah I think that's leftover from before